src/Components/Modal.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from "react"
+
+
+let selectedCounty;
+
+class Modal extends Component {
+    constructor(props) {
+        super(props)
+
+        this.state = {
+            guessedCounty: "",
+        }
+    }
+    
+
+    handleChange = (event) => {
+        event.preventDefault();
+        console.log(event.target.value)
+        // this.setState(() => {
+        //     return {
+        //     guessedCounty: event.target.value
+        //     }
+        // })
+        selectedCounty = `${event.target.value} County`;
+      }
```

Comment:
Rather than having this method here, you could write this in App.js and send it over as props. Then set state with the response of the event.target. 

This would be more of the standard way rather then writing the handler here and exporting both the handler as well as the response from the method